CriteoPublisherSdk/Tests/IntegrationTests/CR_StandaloneBannerFunctionalTests.m

```diff
@@ -68,6 +68,7 @@ - (void)test_givenBannerWithGoodAdUnitId_whenLoadAd_thenAdIsLoadedProperly {
   [self cr_waitForExpectations:@[ viewChecker.adCreativeRenderedExpectation ]];
 }
 
+/* TODO: These can't work with live bidding as injectBidWithExpectedCreativeUrl manipulates cache
```

Comment:
JIRA for TODO?

Reply:
Added a comment on EE-1286 to make sure this is done.